app/scripts/controllers/detect-tokens.js

```diff
@@ -48,7 +69,9 @@ export default class DetectTokensController {
       if (
         contracts[contractAddress].erc20 &&
         !this.tokenAddresses.includes(contractAddress.toLowerCase()) &&
```

Comment:
[Q]: Is there any reason we are not using the isEqualCaseInsensitive here as well?

Reply:
You're probably right that this should be a case insensitive check. Will do now.